src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -61,7 +64,7 @@ private PartModuleFields GetModule(string modName)
             foreach (PartModule mod in Part.Modules)
             {
                 SafeHouse.Logger.Log(string.Format("Does \"{0}\" == \"{1}\"?", mod.moduleName.ToUpper(), modName.ToUpper()));
-                if (String.Equals(mod.moduleName, modName, StringComparison.CurrentCultureIgnoreCase))
+                if (String.Equals(mod.moduleName, modName, StringComparison.OrdinalIgnoreCase))
```

Comment:
This should be a refactor everywhere, or nowhere.  This isn't the only place to use CurrentCultureIgnoreCase, and if we switch we should switch consistently rather than having code that half does it one way and half does it the other.